src/amici_solver.cpp

```diff
@@ -277,13 +260,12 @@ void Solver::getDiagnosisB(const int it, ReturnData *rdata, const BackwardProble
 /**
  * setLinearSolver sets the linear solver for the forward problem
  *
- * @param udata pointer to the user data object
  * @param model pointer to the model object
  */
-void Solver::setLinearSolver(const UserData *udata, Model *model) {
+void Solver::setLinearSolver(Model *model) {
```

Comment:
do we want to call this `initializeLineaSolver`? I would expect that `setLinearSolver` sets `Solver.linsol`